### DISCUSSION TITLE
can it somehow save items in chest even if its spawn protected

### DISCUSSION THREAD

**Discussion by TDLAXD** [mircokroon/minecraft-world-downloader#196] [Q&A]
i foudn this chest on a server and im curious on what it is but i cant open it cuz of spawn protection is it physically possible to get the chest data without opening it?

**Comment by mircokroon** [comment]
Unfortunately not, the server will only send the contents of a chest when its opened.

**Comment by TDLAXD** [comment]
ah ok but what about command blocks if it can save structure block commands
can it save command block commands?

**Comment by mircokroon** [comment]
I'm not sure, I've never tried this. I imagine it won't work but you can give it a try and see what the results are.
